pkg/spdk/server.go

```diff
@@ -466,6 +587,32 @@ func (s *Server) EngineWatch(req *empty.Empty, srv spdkrpc.SPDKService_EngineWat
 	return nil
 }
 
+func (s *Server) EngineReplicaAdd(ctx context.Context, req *spdkrpc.EngineReplicaAddRequest) (ret *empty.Empty, err error) {
```

Comment:
So, this operation is blocking. Will timeout be a concern?

Reply:
Yes. We should use longer timeouts for this API and `ReplicaSnapshotShallowCopy`

Reply:
Should think about an async way later after 1.5.0 to make us support longer rebuild time.